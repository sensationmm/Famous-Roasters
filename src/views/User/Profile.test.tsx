import { render, waitFor } from '@testing-library/react'
import React from 'react'
import { I18nextProvider } from 'react-i18next'
import { MemoryRouter } from 'react-router-dom'
import { i18n } from 'src/config'

import { Profile } from '.'

describe('Profile view', () => {
  it('Renders correctly', async () => {
    const { container } = render(
      <I18nextProvider i18n={i18n}>
        <MemoryRouter initialEntries={['/profile']}>
          <Profile />
        </MemoryRouter>
      </I18nextProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })
})
