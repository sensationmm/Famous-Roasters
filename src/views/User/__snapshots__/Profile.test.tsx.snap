// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile view Renders correctly 1`] = `
<div>
  <div
    class="min-h-screen flex flex-col text-brand-black"
  >
    <div
      class="bg-white z-10"
    >
      <header
        class="relative bg-brand-black"
      >
        <nav
          aria-label="Top"
          class="max-w-7xl mx-auto px-4 md:px-6 xl:px-8"
        >
          <div
            class="h-16 flex items-center justify-between border-b border-brand-black"
          >
            <div
              class="flex-1 flex items-center xl:hidden"
            >
              <button
                class="-ml-2 bg-brand-black p-2 rounded-md text-gray-400"
                data-testid="button-open"
                type="button"
              >
                <span
                  class="sr-only"
                >
                  Open menu
                </span>
                <svg
                  class="undefined fill-brand-black stroke-brand-black"
                  fill="inherit"
                  height="24"
                  viewBox="0 0 24 24"
                  width="24"
                >
                  <path
                    clip-rule="evenodd"
                    d="M3.17188 5H20.0156C20.1102 5 20.1875 5.07734 20.1875 5.17188V6.375C20.1875 6.46953 20.1102 6.54688 20.0156 6.54688H3.17188C3.07734 6.54688 3 6.46953 3 6.375V5.17188C3 5.07734 3.07734 5 3.17188 5ZM3.17188 11.5469H20.0156C20.1102 11.5469 20.1875 11.6242 20.1875 11.7188V12.9219C20.1875 13.0164 20.1102 13.0938 20.0156 13.0938H3.17188C3.07734 13.0938 3 13.0164 3 12.9219V11.7188C3 11.6242 3.07734 11.5469 3.17188 11.5469ZM20.0156 18.0938H3.17188C3.07734 18.0938 3 18.1711 3 18.2656V19.4688C3 19.5633 3.07734 19.6406 3.17188 19.6406H20.0156C20.1102 19.6406 20.1875 19.5633 20.1875 19.4688V18.2656C20.1875 18.1711 20.1102 18.0938 20.0156 18.0938Z"
                    fill="#FFFFFF"
                    fill-rule="evenodd"
                    stroke="#FFFFFF"
                    stroke-width="0.25"
                  />
                </svg>
              </button>
            </div>
            <div
              class="hidden xl:flex-1 xl:block xl:self-stretch"
            >
              <div
                class="h-full flex space-x-10"
              >
                <a
                  class="flex items-center text-white hover:text-coreUI-text-tertiary pb-1.5 duration-500"
                  href="//www.60beans.com/uber-uns"
                >
                  <span
                    class="uppercase text-sm leading-5"
                  >
                    Über uns
                  </span>
                </a>
                <a
                  class="flex items-center text-white hover:text-coreUI-text-tertiary pb-1.5 duration-500"
                  href="//www.60beans.com/roesterei"
                >
                  <span
                    class="uppercase text-sm leading-5"
                  >
                    Unsere Röster
                  </span>
                </a>
                <a
                  class="flex items-center text-white hover:text-coreUI-text-tertiary pb-1.5 duration-500"
                  href="//blog.60beans.com"
                >
                  <span
                    class="uppercase text-sm leading-5"
                  >
                    Blog
                  </span>
                </a>
              </div>
            </div>
            <a
              class="flex h-full items-center pl-2 pr-2"
              href="//www.60beans.de"
            >
              <span
                class="sr-only"
              >
                60beans
              </span>
              <img
                alt="60beans"
                class="h-6"
                src="60beans-light.svg"
              />
            </a>
            <div
              class="flex-1 flow-root flex h-full items-center"
            >
              <div
                class="flex-1 flex h-full items-center justify-end xl:flex-none"
              >
                <div
                  class="hidden h-full xl:flex space-x-8 justify-end"
                >
                  <a
                    class="flex items-center text-white hover:text-coreUI-text-tertiary pb-1.5 duration-500"
                    href="/taste-finder"
                  >
                    <span
                      class="uppercase text-sm leading-5"
                    >
                      Tastefinder
                    </span>
                  </a>
                </div>
                <a
                  class="hidden xl:block ml-10"
                  href="/profile"
                >
                  <svg
                    class="undefined fill-brand-black stroke-brand-black"
                    fill="inherit"
                    height="24"
                    viewBox="0 0 24 25"
                    width="24"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M16 8.66663C16 10.8766 14.21 12.6666 12 12.6666C9.79 12.6666 8 10.8766 8 8.66663C8 6.45663 9.79 4.66663 12 4.66663C14.21 4.66663 16 6.45663 16 8.66663ZM4 18.6666C4 16.0066 9.33 14.6666 12 14.6666C14.67 14.6666 20 16.0066 20 18.6666V20.6666H4V18.6666Z"
                      fill="#FFFFFF"
                      fill-rule="evenodd"
                      stroke="none"
                      stroke-width="none"
                    />
                  </svg>
                </a>
                <div
                  class="ml-4 flow-root xl:ml-10"
                >
                  <button
                    class="relative inline-flex flex-row items-center rounded-full border-2 px-3 py-2 text-sm font-semibold disabled:bg-brand-grey-whisper disabled:text-coreUI-text-secondary disabled:cursor-not-allowed text-brand-black bg-primary border-transparent hover:bg-brand-green-starship active:bg-primary"
                    data-testid="button-shop"
                    role="button"
                  >
                    Shop
                  </button>
                </div>
              </div>
            </div>
          </div>
        </nav>
      </header>
    </div>
    <main
      class="flex flex-col flex-grow w-full items-start bg-white mt-4y"
    >
      <div
        class="w-full border-b border-brand-grey-bombay"
      >
        <div
          class="w-full max-w-7xl mx-auto px-6 xl:px-8 py-8 md:grid md:grid-cols-2 md:gap-20"
        >
          <div>
            <h1
              class="font-syne mb-3 text-3xl leading-10 font-semibold tracking-tight"
            >
              Hallo
               
              asdd
              ,
            </h1>
            <p
              class="text-base leading-6"
            >
              Willkommen in deiner privaten Kaffeeecke. Hier kannst du dein Geschmacksprofil, deine Bestellungen als auch Kontodaten einsehen.
            </p>
          </div>
        </div>
      </div>
      <div
        class="w-full border-b border-brand-grey-bombay"
      >
        <div
          class="w-full max-w-7xl mx-auto px-6 xl:px-8 py-8 grid gap-4 md:grid-cols-2 xl:grid-cols-3"
        >
          <button
            class="relative inline-flex flex-row items-center rounded-full border-2 pl-4 pr-8 py-2 text-base font-bold text-brand-black bg-white border-brand-black hover:border-coreUI-text-tertiary active:border-brand-black pointer-events-none opacity-70"
            data-testid="button-orders"
            disabled=""
            role="button"
          >
            <svg
              class="mr-4 fill-brand-black text-brand-black stroke-brand-black pointer-events-none fill-brand-black stroke-brand-black"
              fill="inherit"
              height="24"
              viewBox="0 0 31 28"
              width="24"
            >
              <path
                clip-rule="evenodd"
                d="M30.7802 10.2277L28.2267 3.64821L15.3901 0L2.5535 3.64821L0 10.2277L1.6579 10.7926V23.5774L15.3903 28L29.1227 23.5774V10.7926L30.7802 10.2277ZM28.9099 9.38404L18.0117 13.1005L16.3124 8.17595L27.236 5.07184L28.9099 9.38404ZM15.39 1.45542L25.1154 4.2196L15.39 6.98355L5.66449 4.2196L15.39 1.45542ZM3.54333 5.0728L14.4669 8.17691L12.7676 13.1015L1.87007 9.38428L3.54333 5.0728ZM3.10315 11.2859L13.6749 14.8907L14.667 12.0143V26.2955L3.10352 22.5706L3.10315 11.2859ZM27.6761 22.5707L16.112 26.2956L16.1122 12.0144L17.1046 14.8908L27.6757 11.2861L27.6761 22.5707Z"
                fill="inherit"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="none"
              />
            </svg>
            Bestellungen
            <svg
              class="absolute right-2 fill-brand-black text-brand-black stroke-brand-black pointer-events-none fill-brand-black stroke-brand-black"
              fill="inherit"
              height="14"
              viewBox="0 0 9 16"
              width="14"
            >
              <path
                clip-rule="evenodd"
                d="M0.999999 1L8 8L1 15"
                fill="none"
                fill-rule="evenodd"
                stroke="inherit"
                stroke-width="2"
              />
            </svg>
          </button>
          <button
            class="relative inline-flex flex-row items-center rounded-full border-2 pl-4 pr-8 py-2 text-base font-bold text-brand-black bg-white border-brand-black hover:border-coreUI-text-tertiary active:border-brand-black pointer-events-none opacity-70"
            data-testid="button-account"
            disabled=""
            role="button"
          >
            <svg
              class="mr-4 fill-brand-black text-brand-black stroke-brand-black pointer-events-none fill-brand-black stroke-brand-black"
              fill="inherit"
              height="24"
              viewBox="0 0 30 30"
              width="24"
            >
              <path
                clip-rule="evenodd"
                d="M5.04687 23.0298C3.26175 20.8494 2.1837 18.0888 2.16435 15.0823L2.16472 15.0822V14.9998C2.16472 7.91007 8.05342 2.14287 15.2934 2.14287C22.5334 2.14287 28.4221 7.91355 28.4221 14.9998C28.4221 18.0382 27.3406 20.8295 25.5353 23.0329C24.2215 19.9059 21.4599 17.5657 18.174 16.6882C19.7034 15.7339 20.7351 14.0713 20.7351 12.1614C20.7351 9.18876 18.2655 6.77288 15.2356 6.77288C12.2057 6.77288 9.73609 9.18876 9.73609 12.1614C9.73609 14.0857 10.7819 15.756 12.3326 16.7076C9.07944 17.6012 6.34969 19.9271 5.04687 23.0298ZM15.2937 29.1C23.2302 29.1 29.6874 22.7784 29.6874 15C29.6874 7.22157 23.2302 0.9 15.2937 0.9C7.35721 0.9 0.9 7.22157 0.9 15C0.9 22.7784 7.35721 29.1 15.2937 29.1ZM15.2937 27.857C11.6703 27.857 8.38703 26.4092 6.00673 24.0764C7.34017 20.2193 11.1009 17.5535 15.2937 17.5535C19.4864 17.5535 23.2467 20.2225 24.5807 24.0764C22.2008 26.4092 18.9173 27.857 15.2937 27.857ZM15.2366 16.3101C12.8992 16.3101 11.0019 14.4489 11.0019 12.1645C11.0019 9.88011 12.8992 8.01887 15.2366 8.01887C17.5741 8.01887 19.4713 9.88011 19.4713 12.1645C19.4711 14.4491 17.5738 16.3101 15.2366 16.3101Z"
                fill="inherit"
                fill-rule="evenodd"
                stroke="inherit"
                stroke-width="0.2"
              />
            </svg>
            Konto
            <svg
              class="absolute right-2 fill-brand-black text-brand-black stroke-brand-black pointer-events-none fill-brand-black stroke-brand-black"
              fill="inherit"
              height="14"
              viewBox="0 0 9 16"
              width="14"
            >
              <path
                clip-rule="evenodd"
                d="M0.999999 1L8 8L1 15"
                fill="none"
                fill-rule="evenodd"
                stroke="inherit"
                stroke-width="2"
              />
            </svg>
          </button>
          <button
            class="relative inline-flex flex-row items-center rounded-full border-2 pl-4 pr-8 py-2 text-base font-bold text-brand-black bg-white border-brand-black hover:border-coreUI-text-tertiary active:border-brand-black"
            data-testid="button-taste-profile"
            role="button"
          >
            <svg
              class="mr-4 fill-brand-black text-brand-black stroke-brand-black fill-brand-black stroke-brand-black"
              fill="inherit"
              height="24"
              viewBox="-1 0 24 24"
              width="24"
            >
              <path
                clip-rule="evenodd"
                d="M14.3625 7.545C14.9238 7.565 15.49 7.74125 16.0088 8.045C17.3313 8.8075 18.1238 10.2337 17.9125 11.9861L17.2088 17.8335C17.0113 19.4797 16.3938 20.6222 15.5001 21.3724C14.605 22.1199 13.4063 22.4849 11.9977 22.4849C10.5889 22.4849 9.39266 22.1199 8.49749 21.3724C7.60248 20.6223 6.98623 19.4799 6.78876 17.8335L6.08501 11.9836C5.83876 9.93857 6.83 8.71732 7.98751 8.04468C9.23 7.35969 10.6263 7.36344 11.6637 8.28968V13.5134C11.6474 13.9784 12.3524 13.9784 12.3362 13.5134L12.3337 8.29096C12.8599 7.80722 13.4824 7.57221 14.1212 7.54472C14.1999 7.54222 14.2799 7.54097 14.3624 7.54472L14.3625 7.545ZM9.31999 5.05374C10.1625 5.07249 11.035 5.29249 11.8537 5.69124C11.9488 5.73749 12.0562 5.73749 12.1488 5.69124C13.79 4.89499 15.6339 4.80625 17.0537 5.63374C17.5812 5.94124 18.6337 6.72625 18.9737 6.97749L17.5687 8.49624C17.2187 8.07749 16.8037 7.7275 16.3487 7.46248C14.9912 6.67122 13.2737 6.60497 11.9986 7.69623C10.7074 6.61873 9.06737 6.64625 7.65112 7.46248C7.19611 7.72498 6.77985 8.07374 6.43237 8.48872L5.02987 6.97745C5.37237 6.7237 6.42488 5.9412 6.95237 5.63371C7.65985 5.2212 8.47486 5.03621 9.31985 5.05371L9.31999 5.05374ZM9.04124 1.5139C9.27624 1.5139 9.50249 1.53765 9.71625 1.5839C10.5713 1.7714 11.2587 2.31388 11.7 3.1464C11.8262 3.3839 12.1662 3.3839 12.2925 3.1464C12.7325 2.31264 13.4212 1.77014 14.2775 1.5839C15.1325 1.3989 16.1725 1.5714 17.305 2.25641L22.0323 5.1214C22.5098 5.41015 22.6761 5.62016 22.7161 5.8239C22.7548 6.0289 22.6711 6.35139 22.3973 6.85264C21.5811 8.35638 20.1923 10.1477 18.6149 11.2952C18.5949 10.4602 18.3537 9.70144 17.9637 9.05644L19.7287 7.1527C19.8637 7.00645 19.8437 6.7752 19.6837 6.65644C19.6837 6.65644 18.1137 5.4752 17.3887 5.0527C15.7749 4.11145 13.765 4.23769 11.9973 5.0527C11.1361 4.65644 10.2248 4.39894 9.32229 4.38019C8.36978 4.36019 7.4373 4.56894 6.60853 5.0527C5.88229 5.4752 4.31352 6.65644 4.31352 6.65644C4.15478 6.77519 4.13478 7.00646 4.26853 7.1527L6.02603 9.04896C5.63352 9.6952 5.39227 10.4589 5.37227 11.2952C3.79601 10.1477 2.40853 8.35644 1.58987 6.85264C1.31737 6.35139 1.23486 6.02889 1.27362 5.8239C1.31237 5.62015 1.48112 5.41014 1.95861 5.1214L6.68245 2.25641C7.53871 1.7414 8.33621 1.51641 9.0412 1.51392L9.04124 1.5139ZM9.03999 0.843884C8.195 0.847634 7.275 1.11638 6.33999 1.68264L1.61359 4.54513C1.07109 4.87512 0.709846 5.22638 0.618582 5.69515C0.527318 6.16392 0.709834 6.62513 1.00732 7.17265C1.94233 8.89515 3.53356 10.9301 5.42364 12.1413L6.11989 17.9151C6.3349 19.6913 7.02489 21.0175 8.06489 21.885C9.10364 22.7562 10.4711 23.1562 11.9961 23.1562C13.5223 23.1562 14.8923 22.7562 15.9311 21.885C16.9699 21.0162 17.6586 19.69 17.8736 17.9151L18.5686 12.1413C20.4611 10.93 22.0538 8.89489 22.9885 7.17265C23.2872 6.62392 23.4685 6.16392 23.3772 5.69515C23.2847 5.22641 22.9234 4.87515 22.3797 4.54513L17.6559 1.68264C16.4084 0.92638 15.1884 0.698892 14.1359 0.928876C13.2396 1.12513 12.5246 1.62888 11.9971 2.39387C11.4696 1.62888 10.7559 1.12513 9.85836 0.928876C9.59461 0.871375 9.3221 0.843877 9.03961 0.843877L9.03999 0.843884Z"
                fill="inherit"
                fill-rule="evenodd"
                stroke="inherit"
                stroke-width="0.75"
              />
            </svg>
            Geschmacksprofil
            <svg
              class="absolute right-2 fill-brand-black text-brand-black stroke-brand-black fill-brand-black stroke-brand-black"
              fill="inherit"
              height="14"
              viewBox="0 0 9 16"
              width="14"
            >
              <path
                clip-rule="evenodd"
                d="M0.999999 1L8 8L1 15"
                fill="none"
                fill-rule="evenodd"
                stroke="inherit"
                stroke-width="2"
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        class="w-full border-b border-brand-grey-bombay"
      >
        <div
          class="w-full max-w-7xl mx-auto px-6 xl:px-8 py-8"
        >
          <div
            class="flex justify-center"
          >
            <svg
              class="w-8 h-8 text-brand-grey-mine animate-spin fill-brand-grey-bombay"
              fill="none"
              role="status"
              viewBox="0 0 100 101"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M100 50.5908C100 78.2051 77.6142 100.591 50 100.591C22.3858 100.591 0 78.2051 0 50.5908C0 22.9766 22.3858 0.59082 50 0.59082C77.6142 0.59082 100 22.9766 100 50.5908ZM9.08144 50.5908C9.08144 73.1895 27.4013 91.5094 50 91.5094C72.5987 91.5094 90.9186 73.1895 90.9186 50.5908C90.9186 27.9921 72.5987 9.67226 50 9.67226C27.4013 9.67226 9.08144 27.9921 9.08144 50.5908Z"
                fill="currentColor"
              />
              <path
                d="M93.9676 39.0409C96.393 38.4038 97.8624 35.9116 97.0079 33.5539C95.2932 28.8227 92.871 24.3692 89.8167 20.348C85.8452 15.1192 80.8826 10.7238 75.2124 7.41289C69.5422 4.10194 63.2754 1.94025 56.7698 1.05124C51.7666 0.367541 46.6976 0.446843 41.7345 1.27873C39.2613 1.69328 37.813 4.19778 38.4501 6.62326C39.0873 9.04874 41.5694 10.4717 44.0505 10.1071C47.8511 9.54855 51.7191 9.52689 55.5402 10.0491C60.8642 10.7766 65.9928 12.5457 70.6331 15.2552C75.2735 17.9648 79.3347 21.5619 82.5849 25.841C84.9175 28.9121 86.7997 32.2913 88.1811 35.8758C89.083 38.2158 91.5421 39.6781 93.9676 39.0409Z"
                fill="currentFill"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        class="w-full border-b border-brand-grey-bombay"
      >
        <div
          class="w-full max-w-7xl mx-auto px-6 xl:px-8 py-8"
        >
          <h2
            class="mb-3 text-2xl leading-8 font-semibold"
          >
            Mehr Produkte für dich
          </h2>
          <p
            class="mb-3 text-base leading-6"
          >
            Genieße deinen Lieblingskaffee in vollen Zügen – entdecke dafür unseren Kaffeezubehör.
          </p>
          <div
            class="grid gap-2 grid-cols-1 md:grid-cols-2 xl:grid-cols-3"
          >
            <a
              href="/product/7655228866776"
            >
              <div
                class="flex pt-8"
              >
                <img
                  alt="Espresso Raritäten Set"
                  class="hidden"
                  src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
                />
                <div
                  class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
                >
                  <div
                    class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
                  >
                    <img
                      alt="Espresso Raritäten Set"
                      class="w-28 max-h-28"
                      src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
                    />
                    <div
                      class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
                    >
                      <span
                        class="text-xs leading-4"
                      >
                        Decaf
                      </span>
                    </div>
                  </div>
                </div>
                <div
                  class="absolute"
                >
                  <div
                    class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
                  >
                    <div
                      class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
                    >
                      <div
                        class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="flex flex-col w-full justify-center pl-4"
                >
                  <div
                    class="text-base leading-6 font-semibold"
                  >
                    Espresso Raritäten Set
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    60beans
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    Kolumbien
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    Filter
                  </div>
                  <div
                    class="flex items-baseline"
                  >
                    <span
                      class="mr-1 text-base leading-6 font-semibold"
                    >
                      19,99 €
                    </span>
                    <span
                      class="text-coreUI-text-secondary text-xs leading-4"
                    >
                      (
                      10,00 €
                      /kg)
                    </span>
                  </div>
                </div>
              </div>
            </a>
            <a
              href="/product/7655228866776"
            >
              <div
                class="flex pt-8"
              >
                <img
                  alt="Espresso Raritäten Set"
                  class="hidden"
                  src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
                />
                <div
                  class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
                >
                  <div
                    class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
                  >
                    <img
                      alt="Espresso Raritäten Set"
                      class="w-28 max-h-28"
                      src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
                    />
                    <div
                      class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
                    >
                      <span
                        class="text-xs leading-4"
                      >
                        Decaf
                      </span>
                    </div>
                  </div>
                </div>
                <div
                  class="absolute"
                >
                  <div
                    class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
                  >
                    <div
                      class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
                    >
                      <div
                        class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="flex flex-col w-full justify-center pl-4"
                >
                  <div
                    class="text-base leading-6 font-semibold"
                  >
                    Espresso Raritäten Set
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    60beans
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    Kolumbien
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    Filter
                  </div>
                  <div
                    class="flex items-baseline"
                  >
                    <span
                      class="mr-1 text-base leading-6 font-semibold"
                    >
                      19,99 €
                    </span>
                    <span
                      class="text-coreUI-text-secondary text-xs leading-4"
                    >
                      (
                      10,00 €
                      /kg)
                    </span>
                  </div>
                </div>
              </div>
            </a>
            <a
              href="/product/7655228866776"
            >
              <div
                class="flex pt-8"
              >
                <img
                  alt="Espresso Raritäten Set"
                  class="hidden"
                  src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
                />
                <div
                  class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
                >
                  <div
                    class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
                  >
                    <img
                      alt="Espresso Raritäten Set"
                      class="w-28 max-h-28"
                      src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
                    />
                    <div
                      class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
                    >
                      <span
                        class="text-xs leading-4"
                      >
                        Decaf
                      </span>
                    </div>
                  </div>
                </div>
                <div
                  class="absolute"
                >
                  <div
                    class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
                  >
                    <div
                      class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
                    >
                      <div
                        class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="flex flex-col w-full justify-center pl-4"
                >
                  <div
                    class="text-base leading-6 font-semibold"
                  >
                    Espresso Raritäten Set
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    60beans
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    Kolumbien
                  </div>
                  <div
                    class="text-coreUI-text-secondary text-base leading-6"
                  >
                    Filter
                  </div>
                  <div
                    class="flex items-baseline"
                  >
                    <span
                      class="mr-1 text-base leading-6 font-semibold"
                    >
                      19,99 €
                    </span>
                    <span
                      class="text-coreUI-text-secondary text-xs leading-4"
                    >
                      (
                      10,00 €
                      /kg)
                    </span>
                  </div>
                </div>
              </div>
            </a>
          </div>
          <div
            class="flex justify-center pt-14"
          >
            <button
              class="relative inline-flex flex-row items-center rounded-full border-2 justify-center px-4 py-2 text-base font-bold text-brand-black bg-white border-brand-black hover:border-coreUI-text-tertiary active:border-brand-black w-full md:max-w-xs md:mx-auto"
              role="button"
            >
              Zum Kaffeezubehör 
            </button>
          </div>
        </div>
      </div>
      <div
        class="w-full border-b border-brand-grey-bombay"
      >
        <div
          class="w-full max-w-7xl mx-auto px-6 xl:px-8 py-8 pb-14"
        >
          <h2
            class="text-2xl leading-8 font-semibold"
          >
            Kaffeewissen für dich
          </h2>
          <div
            class="grid w-full md:grid-cols-2"
          >
            <div
              class="grid grid-cols-2 grid-cols-[150px_1fr] gap-4 px-0 py-6 border-b border-brand-grey-bombay md:max-w-sm md:border-0"
            >
              <a
                href="/catalogue?vendor=Nomad"
              >
                <img
                  class="w-full"
                  src="coffee-taste.png"
                />
              </a>
              <div
                class="flex flex-col justify-center"
              >
                <p
                  class="mb-4 text-base leading-6"
                >
                  NOMAD Coffee is our latetest roaster in the catalog!
                </p>
                <button
                  class="relative inline-flex flex-row items-center rounded-full border-2 pl-3 pr-8 py-2 text-xs font-semibold text-white bg-brand-black border-transparent hover:bg-coreUI-text-secondary active:bg-brand-black"
                  data-testid="button-vendor=Nomad"
                  role="button"
                >
                  Produkte erkunden
                  <svg
                    class="absolute right-2 !fill-white !text-white !stroke-white fill-brand-black stroke-brand-black"
                    fill="inherit"
                    height="14"
                    viewBox="0 0 15 13"
                    width="14"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M8.83333 1.83301L13.5 6.49967M13.5 6.49967L8.83333 11.1663M13.5 6.49967L1.5 6.49967"
                      fill="inherit"
                      fill-rule="evenodd"
                      stroke="inherit"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                </button>
              </div>
            </div>
            <div
              class="grid grid-cols-2 grid-cols-[150px_1fr] gap-4 px-3 py-6 md:max-w-sm"
            >
              <a
                href="/catalogue?origin=BU"
              >
                <img
                  class="w-full"
                  src="myanmar.png"
                />
              </a>
              <div
                class="flex flex-col justify-center"
              >
                <p
                  class="mb-4 text-base leading-6"
                >
                  Discover new complex origins: Myanmar!
                </p>
                <button
                  class="relative inline-flex flex-row items-center rounded-full border-2 pl-3 pr-8 py-2 text-xs font-semibold text-white bg-brand-black border-transparent hover:bg-coreUI-text-secondary active:bg-brand-black"
                  data-testid="button-origin=BU"
                  role="button"
                >
                  Produkte erkunden
                  <svg
                    class="absolute right-2 !fill-white !text-white !stroke-white fill-brand-black stroke-brand-black"
                    fill="inherit"
                    height="14"
                    viewBox="0 0 15 13"
                    width="14"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M8.83333 1.83301L13.5 6.49967M13.5 6.49967L8.83333 11.1663M13.5 6.49967L1.5 6.49967"
                      fill="inherit"
                      fill-rule="evenodd"
                      stroke="inherit"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="bg-brand-black text-base text-white py-10 z-30"
    >
      <div
        class="w-full max-w-7xl mx-auto mb-10 flex flex-col md:flex-row md:items-center justify-start mx-auto px-8 xl:px-10"
      >
        <div
          class="mb-2 md:mb-0 md:mr-2"
        >
          <svg
            fill="none"
            height="157"
            viewBox="0 0 400 400"
            width="157"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="200"
              cy="200"
              fill="#D4FF81"
              r="200"
            />
            <path
              clip-rule="evenodd"
              d="M266 225.882C266 267.294 243.14 288 214.739 288C196.728 288 183.336 279.488 177.332 266.604V285.239H136V112H177.332V182.86C184.259 171.357 196.728 161.694 214.97 161.694C243.14 161.694 266 182.86 266 224.042V225.882ZM231.006 225.964C231.006 240.559 219.132 252.391 204.483 252.391C189.835 252.391 177.96 240.559 177.96 225.964C177.96 211.369 189.835 199.538 204.483 199.538C219.132 199.538 231.006 211.369 231.006 225.964ZM251 156C263.15 156 273 146.15 273 134C273 121.85 263.15 112 251 112C238.85 112 229 121.85 229 134C229 146.15 238.85 156 251 156Z"
              fill="#212121"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="px-4 text-5xl leading-none"
        >
          Guten Kaffee besser finden.
        </div>
      </div>
      <div
        class="w-full max-w-7xl mx-auto grid gap-2 gap-y-10 grid-cols-1 md:grid-cols-2 mx-auto px-12 xl:px-14"
      >
        <div
          class="order-2 md:order-1"
        >
          <ul>
            <li
              class="mt-0"
            >
              <a
                href="/catalogue"
              >
                Shop
              </a>
            </li>
            <li
              class="mt-6"
            >
              <a
                href="/taste-finder"
              >
                Tastefinder
              </a>
            </li>
            <li
              class="mt-6"
            >
              <a
                href="//www.60beans.de/uber-uns"
              >
                Über uns
              </a>
            </li>
            <li
              class="mt-6"
            >
              <a
                href="//www.notion.so/Datenschutzhinweise-ad5b45261a964166a74883d2770e28ea"
              >
                Datenschutz-Bestimmungen
              </a>
            </li>
            <li
              class="mt-6"
            >
              <a
                href="//slender-petunia-c3c.notion.site/Imprint-33b18fd9e469471ab768eef8155da968"
              >
                Impressum
              </a>
            </li>
          </ul>
        </div>
        <div
          class="order-1 md:order-2"
        >
          <div>
            <div>
              <span
                class="font-syne text-3xl leading-10 font-semibold tracking-tight"
              >
                Kundensupport
              </span>
            </div>
            <div
              class="mt-6"
            >
              <span
                class="text-lg leading-7"
              >
                Fragen? Kommentare? Bedenken? Oder möchtest du uns einfach ein paar schöne Worte senden? Bitte klicke einfach unten rechts auf die Sprechblasen. Unser Team steht dir dort gerne zur Verfügung.
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full max-w-7xl mx-auto text-center mt-10 mx-auto px-12 xl:px-14"
      >
        <span
          class="text-center text-base leading-6"
        >
          © 2022 60beans. Alle Rechte vorbehalten.
        </span>
        <br />
        <span
          class="text-center text-base leading-6"
        >
          Hergestellt mit ❤️ in Berlin.
        </span>
      </div>
    </footer>
  </div>
</div>
`;
